my_list.rb

```diff
@@ -0,0 +1,30 @@
+require_relative 'my_enumerables'
+class MyList
+  include MyEnumerable
+
+  def initialize(*args)
+    @list = args
+  end
+
+  def each(&block)
+    @list.each(&block)
+  end
+end
```

Comment:
- good job on creating the MyList class 👍 however the `#each` method you created here should be created by yourselves not using the existing `#each` enumerable.

Reply:
Hello, @mikethreels 👋

Thank you for reviewing our work. I have a question regarding the required changes that you have proposed. Would implementing a `.map` method do for creating the `#each` method in the `MyList` class?

Or maybe, should we implement it using a `for` loop and the `@list.length`?

Thanks in advance for your help! Cheers and Happy coding to you too! 🤓

Reply:
Hi @rpire 👋 

Try implementing it using a loop `for` or `while` using the `@list.length`.
looking at your other methods this should be a walk in the park for you 🥇 

You're doing great.
let me know if you have any additional questions!

Reply:
Greetings again, @mikethreels 👋

About my last question, I have decided to implement a `while` loop. Using the `.map` method was the same as using the `.each` method, and using a `for` loop would give Linter errors.

Required changes completed! 🙌